modules/http/src/main/java/com/spotify/ffwd/http/HttpException.kt

```diff
@@ -18,17 +18,8 @@
  * -/-/-
  */
 
-package com.spotify.ffwd.http;
+package com.spotify.ffwd.http
 
-import io.netty.handler.codec.http.HttpResponseStatus;
-import lombok.Data;
+import io.netty.handler.codec.http.HttpResponseStatus
 
-@Data
-public class HttpException extends RuntimeException {
-    private final HttpResponseStatus status;
-
-    public HttpException(final HttpResponseStatus status) {
-        super(status.reasonPhrase());
-        this.status = status;
-    }
-}
+class HttpException(val status: HttpResponseStatus?) : RuntimeException(status?.reasonPhrase())
```

Comment:
The Java version looks like it was non-nullable (or at least it would throw a null-pointer exception on the super call), so I suggest making the Kotlin param non-nullable to match it.

Reply:
So what your class says now is that it takes a nullable HttpResponseStatus as the constructor arg, then supers a RuntimeException with a non-null check. You can remove the !! and change `HttpResponseStatus?` to `HttpResponseStatus` and the null check will happen at compile time.

Reply:
fixed.